locale/en_US/translations.ts

```diff
@@ -1114,5 +1114,10 @@
             <translation>Set how many seconds before the end of an episode the Next Episode button should appear. Set to 0 to disable.</translation>
             <extracomment>Settings Menu - Description for option</extracomment>
         </message>
+        <message>
```

Comment:
```suggestion
		<message>
			<source>Direct playing</source>
			<translation>Direct playing</translation>
        <message>
```

I didn't see `Direct playing` in the translation file either